fluentd/configs.d/openshift/output-es-config.conf

```diff
@@ -3,14 +3,16 @@
       host "#{ENV['ES_HOST']}"
       port "#{ENV['ES_PORT']}"
       scheme https
-      index_name ${record['kubernetes_namespace_name']}.${record['kubernetes_namespace_id']}.${Time.at(time).getutc.strftime(@logstash_dateformat)}
+      index_name project.${record['kubernetes']['namespace_name']}.${record['kubernetes']['namespace_id']}.${Time.parse(record['@timestamp']).getutc.strftime(@logstash_dateformat)}
       user fluentd
       password changeme
 
       client_key "#{ENV['ES_CLIENT_KEY']}"
       client_cert "#{ENV['ES_CLIENT_CERT']}"
       ca_file "#{ENV['ES_CA']}"
 
+      type_name com.redhat.viaq.common
```

Comment:
I fixed it as well as check-logs.sh to adjust the doc type.  Plus added the nil check for record.Fields.Message to check-logs.go to avoid the panic by "index out of range".